Detector/DetFCChhTrackerSimple/src/GenericEndcap_Geo.cpp

```diff
@@ -0,0 +1,125 @@
+
+#include "DetCommon/DetUtils.h"
+#include "DD4hep/DetFactoryHelper.h"
+
+using DD4hep::Geometry::Volume;
+using DD4hep::Geometry::DetElement;
+using DD4hep::XML::Dimension;
+using DD4hep::Geometry::PlacedVolume;
+
+namespace det {
+/**
+Factory for a configurable, generic tracker endcap.
```

Comment:
I wrote a short comment in the xml: https://github.com/vvolkl/FCCSW/blob/generic_tracker/Detector/DetFCChhTrackerSimple/compact/Tracker.xml#L46